### ISSUE TITLE
IRI logs fill up disk space and cause the server to crash

### ISSUE THREAD

**Issue by rajivshah3** [iotaledger/iri#764]
### Bug description
Reporting this on behalf of Blu Jay Sidewinder and Typpi

Both report that at one point, IRI caused the logs to fill up all available disk space (logs became over 100GB in size). This causes the system to crash so that it is totally inaccessible, even through `ssh`.

### Hardware Spec

Typpi: 32 cores, 128GB RAM
Blu Jay Sidewinder: 2 cores, 4GB RAM

### Errors
Will post logs from both nodes if possible

**Comment by footloosejava** [comment]
I suggest that if the `processed, queued, requested` line that prints every few seconds has nothing new (ie. same values as last entry), then just print a dot or something.

There is no need to see it if nothing has changed. Otherwise every thread cycle is just keeps printing the same info ... every ... time ...

That should get rid of some bloat.

Now question: Is the logs being teed to a file (piped or appended)? Because if that is done by the user then it is out of control of the IRI.

**Comment by GalRogozinski** [comment]
As mentioned above IRI doesn't write a log file by default. So if @rajivshah3 agrees I will close this issue.

**Comment by rajivshah3** [comment]
@GalRogozinski I don't think this issue occurred again and I haven't heard anything from either of the users who experienced this so I think it can be closed

**Comment by rajivshah3** [comment]
This seems to relate to #811
